ui/app/src/components/BrowseS3Dialog/BrowseExternalAssetDialog.tsx

```
@@ -0,0 +1,402 @@
+}) {
+	const label = folder?.name ?? path ?? '/';
+	return (
+		<>
+			<TreeItem itemId={path} label={<Box onClick={(e) => onFolderClick(e, path)}>{label}</Box>}>
+				{foldersByPath[path]?.map((folder) => (
+					<ExternalAssetFoldersTreeView
+						foldersByPath={foldersByPath}
+						path={folder.path}
+						folder={folder}
+						onFolderClick={onFolderClick}
+					/>
```

Biome (2.4.7)

[error] Missing key property for this element in iterable.

(lint/correctness/useJsxKeyInIterable)